debian/chromium-browser.sh.in

```diff
@@ -7,11 +7,25 @@
 #  Fabien Tassin <fta@sofaraway.org>
 # License: GPLv2 or later
 
-ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
+skip_parental_controls_check=0
+for i in "$@"; do
+  case "$i" in
+    --pack-extension|--pack-extension=* )
+      # Lets skip parental controls check if packing extensions
+      skip_parental_controls_check=1
+      break ;;
+    * )
+      ;;
+  esac
+done
 
-if ! malcontent-client --quiet check --no-interactive ${ABSOLUTE_PATH}; then
-  echo "error: $ABSOLUTE_PATH is blacklisted for the current user" >&2
-  exit 1
+if [ $skip_parental_controls_check -eq 0 ]; then
+  ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
```

Comment:
I know this is existing code but what does this do that something like `realpath "${BASH_SOURCE[0]}"` doesn't?

Reply:
I've seem this enough times that I didnt stop to think why tbh :D - it may be related to symlinks resolution. It also seems to have something to do with OSX support (not that we care about). 

Do you want an update here? tbh I would keep as is as it just works.

Reply:
OK